### PR TITLE
frontend/feat/#3430-rideConfirmation message

### PR DESCRIPTION
**Pull request by MercyQueen**
## Type of Change

- [ ] Bugfix
- [x] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

### CODE REVIEW

File: Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs [hunk 1]
```diff
@@ -635,6 +636,15 @@ eval TerminateApp state = do
   pure $ terminateApp state.props.currentStage true
   continue state
 
+eval (CountDown seconds id status timerID) state = do
```
Comment (ashishsingh101) [inline Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs:639]: can we rename action name according to its behaviour

File: Frontend/ui-customer/src/Helpers/Storage.purs [hunk 2]
```diff
@@ -80,6 +80,7 @@ data KeyStore
   | DISABILITY_UPDATED
   | DISABILITY_POPUP_TRIGGERED
   | BUNDLE_TIME_OUT
+  | NOTIFIEDCUSTOMER
```
Comment (ashishsingh101) [inline Frontend/ui-customer/src/Helpers/Storage.purs:83]: can we make it snake case

File: Frontend/android-native/app/src/user/jatriSaathi/assets/juspay/v1-configuration.js [hunk 3]
```diff
@@ -242,6 +242,8 @@ window.getMerchantConfig = function () {
   , "callOptions" : ["ANONYMOUS"]
   , "showNearByDrivers" : true
   , "showDisabilityBanner" : false
+  , "notifyRideConfirmation" : true
```
Comment (Vignesh-772) [inline Frontend/android-native/app/src/user/jatriSaathi/assets/juspay/v1-configuration.js:245]: put these two values inside a object

File: Frontend/android-native/app/src/user/jatriSaathi/assets/juspay/v1-configuration.js [hunk 4]
```diff
@@ -242,6 +242,8 @@ window.getMerchantConfig = function () {
...
+  , "autoGeneratedText" : "Nomoshkar, I'll arrive in about "
```
Comment (ashishsingh101) [inline Frontend/android-native/app/src/user/jatriSaathi/assets/juspay/v1-configuration.js:246]: can we make parent object for these fields

File: Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/View.purs [hunk 5]
```diff
@@ -142,11 +142,16 @@ screen initialState =
                 _ <- pure $ setValueToLocalStore SESSION_ID (generateSessionId unit)
                 _ <- pure $ removeAllPolylines ""
                 _ <- pure $ enableMyLocation true
+                _ <- pure $ setValueToLocalStore NOTIFIEDCUSTOMER "false"
```
Comment (Vignesh-772) [inline Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/View.purs:145]: change to `NOTIFIED_CUSTOMER`

File: Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs [hunk 6]
```diff
@@ -635,6 +636,15 @@ eval TerminateApp state = do
   pure $ terminateApp state.props.currentStage true
   continue state
 
+eval (CountDown seconds id status timerID) state = do 
+  if status == "EXPIRED" then do
```
Comment (Vignesh-772) [inline Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs:640]: Can you confirm what is expected behaviour here.

**Review by ashishsingh101 — CHANGES_REQUESTED** [review]

File: Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs (hunk 6 above)
Comment (MercyQueen) [reply Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs:640]: once the 5sec timer expires then we have to show this message

**Review by Vignesh-772 — DISMISSED** [review]

**Review by ashishsingh101 — APPROVED** [review]

**Review by ashishsingh101 — DISMISSED** [review]

**Review by ashishsingh101 — APPROVED** [review]

**Pull request merged**